lib/src/main/java/uk/gov/di/ipv/core/library/service/ConfigService.java

```diff
@@ -182,6 +189,44 @@ public String getCriPrivateApiKey(String criId) {
         }
     }
 
+    public String getActiveConnection(String credentialIssuerId) {
+        return getSsmParameter(
+                String.format(
+                        "%s/%s/activeConnection",
+                        getEnvironmentVariable(CREDENTIAL_ISSUERS_CONFIG_PARAM_PREFIX),
+                        credentialIssuerId));
+    }
+
+    public boolean isUnavailable(String credentialIssuerId) {
```

Comment:
I think, we can add generic method for all 3 methods and get this value unavailable, allowedsharedAttributes, enabled.. from second method param.